src/common/ceph_crypto.h

```diff
@@ -98,7 +98,9 @@ namespace ceph::crypto::ssl {
   public:
     HMAC (const EVP_MD *type, const unsigned char *key, size_t length)
       : mpType(type) {
-      ::memset(&mContext, 0, sizeof(mContext));
+      // the strict FIPS zeroization doesn't seem to be necessary here.
+      // just in the case.
+      ::ceph::crypto::zeroize_for_security(&mContext, sizeof(mContext));
```

Comment:
Yes, this one really shouldn't be zeroize_for_security.  That context hasn't yet got any nasty secrets in it, so no need to sanitize.